snowpack/src/commands/install.ts

```diff
@@ -342,17 +348,17 @@ ${colors.dim(
         namedExports: true,
       }),
       rollupPluginCss(),
+      rollupPluginReplace(
```

Comment:
It's not clear to me whether we should restrict the replacement to specific file types (e.g., `js`, `jsx`, `ts`, `tsx`) or let it run on all files.  I think `rollupPluginNodeProcessPolyfill` runs on all files but will skip with a warning ones it can't parse?

Reply:
Lets keep existing behavior for now, I haven't heard this complaint yet so far